action.yml

```
@@ -26,6 +26,14 @@ inputs:
     description: 'Skip automated testing such as pytest'
     required: false
     default: false
+  prebuild-package-path:
+    description: 'Path to brebuild app zip'
+    required: false
+    default: ''
+  ignored-files:
+    description: 'File patterns to skip zip'
```

```suggestion
  ignored-paths:
    description: 'Path patterns to skip zip, separated by space'
```

Since we want to ignore both files and folders, can it be changed to `ignored-paths` instead?